product_docs/docs/pem/9/certificates/replacing_ssl_certificates.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: "Regenerating the server self-signed SSL certificates"
```

Comment:
It might be useful to add something on this page to explain which of the steps are still relevant if you are using externally provided certificates